src/store/modules/rate/index.js

```diff
@@ -41,22 +41,26 @@ const actions = {
         })
     })
   },
-  getHistoryRates ({ commit }, { date, coin }) {
+  getHistoryRates ({ state, commit }, { date, coin }) {
```

Comment:
Return not all rates for timestamp,
Instead, return rate for timestamp for `{ADM}/{fiatFromSettings}`